src/gss_ntlmssp.h

```diff
@@ -40,6 +40,13 @@
 #define NTLMSSP_CTX_FLAG_SPNEGO_CAN_MIC 0x02 /* SPNEGO asks for MIC */
 #define NTLMSSP_CTX_FLAG_AUTH_WITH_MIC  0x04 /* Auth MIC was created */
 
+struct gssntlm_name_attribute {
+    const char *attr_name; /* Read-only static string, should not be released */
```

Comment:
I am not sure this is exactly a good idea, seem like an optimization that won't buy us much in terms of memory usage but could byte us in the future (in terms of reachitecting) should we need dynamically allocated names.
Howerver I am ok keeping this static given it has already been implemented.

Reply:
Attribute names are pre-defined and from both sides (library and the caller side) the names are always static. Currently I don't see a reason why do we need to copy this name to dynamic memory once it's always in static memory of the library (even if the caller uses dynamic memory, we always store a pointer to OUR static memory).
'const' modifier and the comment should make it clear.

Reply:
ok